packages/synapse-react-client/src/components/UserCard/UserBadge.tsx

```diff
@@ -30,6 +30,8 @@ export type UserBadgeProps = {
   showFullName?: boolean
 
   className?: string
+
+  isModerator?: boolean
```

Comment:
nit: Be a bit more explicit about what this prop does
```suggestion
  showModeratorBadge?: boolean
```